packages/deno/src/integrations/globalhandlers.ts

```diff
@@ -87,10 +87,16 @@ function installGlobalErrorHandler(): void {
     data.preventDefault();
     isExiting = true;
 
-    void flush().then(() => {
-      // rethrow to replicate Deno default behavior
-      throw error;
-    });
+    void flush().then(
+      () => {
+        // rethrow to replicate Deno default behavior
+        throw error;
+      },
+      () => {
+        // rethrow to replicate Deno default behavior
+        throw error;
+      },
```

Comment:
So my thought process here was that we are just intercepting a global error here, meaning we should always rethrow because we want the process to crash with the original error. If `flush` threw, we would be crashing the process because sending the event failed - which I don't think is what we want. With this change, even if flush throws, we will still be crashing because of the original error. Does this make sense?

Reply:
@timfish You think this is fine?

Reply:
Yep, this should be fine.

Reply:
ahh yes, good point, makes sense!